test/components/accordion/accordion.test.js

```diff
@@ -0,0 +1,57 @@
+import '../accordion.js';
+import { expect, fixture, html } from '@brightspace-ui/testing';
+
+const accordionCollapseFixture = html`
+	<d2l-labs-accordion>
+		<d2l-labs-accordion-collapse title="Item 1">Test</d2l-labs-accordion-collapse>
+		<d2l-labs-accordion-collapse title="Item 2">Test</d2l-labs-accordion-collapse>
+		<d2l-labs-accordion-collapse title="Item 3">Test</d2l-labs-accordion-collapse>
+	</d2l-labs-accordion>
+`;
+const accordionCollapseFixtureLabel = html`
+	<d2l-labs-accordion>
+		<d2l-labs-accordion-collapse label="Item 1">Test</d2l-labs-accordion-collapse>
+		<d2l-labs-accordion-collapse label="Item 2">Test</d2l-labs-accordion-collapse>
+		<d2l-labs-accordion-collapse label="Item 3">Test</d2l-labs-accordion-collapse>
+	</d2l-labs-accordion>
+`;
+
+describe('d2l-labs-accordion', () => {
+
+	it('initial select state', async() => {
+		const myEl = await fixture(accordionCollapseFixture);
+		expect(myEl.selected).to.not.exist;
+		expect(myEl.items[0].opened).to.be.false;
+		expect(myEl.items[1].opened).to.be.false;
+		expect(myEl.items[2].opened).to.be.false;
+		expect(myEl.items[0].shadowRoot.querySelector('.collapse-title').title).to.equal('');
+		expect(myEl.items[1].shadowRoot.querySelector('.collapse-title').title).to.equal('');
+		expect(myEl.items[2].shadowRoot.querySelector('.collapse-title').title).to.equal('');
+	});
+
+	it('state change', async() => {
+		const myEl = await fixture(accordionCollapseFixture);
+		expect(myEl.items[0].opened).to.be.false;
+		expect(myEl.items[1].opened).to.be.false;
+		expect(myEl.items[2].opened).to.be.false;
+		myEl.selected = 1;
+		expect(myEl.items[0].opened).to.be.false;
+		expect(myEl.items[1].opened).to.be.true;
+		expect(myEl.items[2].opened).to.be.false;
+	});
+
+	it('trigger test', async() => {
+		const myEl = await fixture(accordionCollapseFixture);
+		expect(myEl.items[0].opened).to.be.false;
+		myEl.items[0].shadowRoot.querySelector('#trigger').click();
```

Comment:
Probably best to await this
```suggestion
		await myEl.items[0].shadowRoot.querySelector('#trigger').click();
```

Reply:
Oh, wait sorry I was thinking of this being `await clickElem(myEl...)`. I guess this fine.